services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_compose_egress_config.py

```diff
@@ -197,12 +221,17 @@ def _get_egress_proxy_service_config(
         "command": command,
         "networks": {
             # allows the proxy to access the internet
-            swarm_network_name: None,
+            network_with_internet: None,
             # allows containers to contact proxy via these aliases
             _get_egress_proxy_network_name(egress_proxy_name): {
                 "aliases": list(network_aliases)
             },
         },
+        # NOTE: there as some issues with ho DNS in house
```

Comment:
you might want to change that comment. in house? like in open source house?

Reply:
removed, no longer used.